nodejs/advanced/Sentiment-demo-ui/tsconfig.app.json

```diff
@@ -1,14 +1,15 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
Is this kind of comment valid in a json file?

Reply:
think so. autogenerated file.